packages/in_app_purchase/in_app_purchase/example/integration_test/in_app_purchase_test.dart

```diff
@@ -4,12 +4,14 @@
 
 import 'package:flutter_test/flutter_test.dart';
 import 'package:in_app_purchase/in_app_purchase.dart';
+import 'package:in_app_purchase_android/in_app_purchase_android.dart';
 import 'package:integration_test/integration_test.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   testWidgets('Can create InAppPurchase instance', (WidgetTester tester) async {
+    InAppPurchaseAndroidPlatformAddition.enablePendingPurchases();
```

Comment:
Why is this necessary for a placeholder test?

Reply:
The constructor of `InAppPurchaseAndroidPlatformAddition` has an assert that requires this to be called. This is triggered when a new singleton is created via the instance getter.

Reply:
That doesn't seem ideal; can you file an issue about this and annotate the line with a TODO referencing it?

Also, don't we need to platform-gate this call?